UPGRADE-14.0.md

```diff
@@ -769,6 +769,21 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   add Category ID to CategoryHierarchyItem ([#2962](https://github.com/shopsys/shopsys/pull/2962))
     -   see #project-base-diff to update your project
 -   add display timezone to FE API SettingsQuery ([#2977](https://github.com/shopsys/shopsys/pull/2977))
+    -   `Shopsys\FrameworkBundle\Component\DateTimeHelper\DateTimeHelper` class was changed:
+        -   `UTC_TIMEZONE` constant was removed
+        -   `TIME_REGEX` constant was removed
+        -   `convertDatetimeStringFromDisplayTimeZoneToUtc()` method was removed
+        -   `convertDateTimeFromUtcToDisplayTimeZone()` method was removed
```

Comment:
everything was added in this version, so I think the upgrade notes are not necessary

Reply:
ah, good catch :ok_hand: fixed in f1a124537